crypto/ledger.go

```diff
@@ -0,0 +1,20 @@
+// +build ledger
+
+package crypto
+
+import (
+	ledger "github.com/zondax/ledger-goclient"
+)
+
+// If ledger support (build tag) has been enabled, automically attempt to load
+// and set the ledger device, ledgerDevice, if it has not already been set.
+func init() {
+	if ledgerDevice == nil {
```

Comment:
Why do we need to check this - `init()` will only be called once?

Reply:
Yes, `init` will only be called once. Alternatively, you could wrap it in a `once.Do(...)` call.

Reply:
Nvm, I see you're point. The old code was this way -- I'll remove it 👍